src/util/misc.py

```diff
@@ -39,8 +39,19 @@
 _logger = logging.getLogger(__name__)
 
 
+class Sentinel:
+    # waiting PEP661...
+    __slots__ = ("name",)
+
+    def __init__(self, name):
+        self.name = name
+
+    def __repr__(self):
+        return self.name
```

Comment:
nice to have: in debugging sessions it's nice if this is explicitly printed as "Sentinel"
```suggestion
        return "Sentinel<{}>".format(self.name)
```

Reply:
Actually, it's for having it display nicely in the documentation: https://runbot131.odoo.com/runbot/static/build/77170216-master/logs/build/html/developer/reference/upgrades/upgrade_utils.html#odoo.upgrade.util.pg.ColumnList.using

<img width="829" alt="image" src="https://github.com/user-attachments/assets/57cabea8-3cc6-4b42-9607-26dca07ab2a0" />